app/helpers/application_helper.rb

```diff
@@ -49,11 +49,16 @@ def single_relationship_link(record, table_name, property_name = nil)
     name = ui_lookup(:table => table_name.to_s)
     if role_allows(:feature => "#{table_name}_show") && !ent.nil?
       out = content_tag(:li) do
+        link_params = if restful_routed?(ent)
+                        polymorphic_path(ent)
+                      else
+                        {:controller => table_name, :action => 'show', :id => ent.id.to_s}
+                      end
```

Comment:
I though this was the syntactic sugar.. I can change to a ternary operator but it will produce a 136 chars long line.